quaistats/quaistats.go

```diff
@@ -55,6 +56,10 @@ const (
 
 	// reportInterval is the time interval between two reports.
 	reportInterval = 15
+
+	alpha               = 8
+	tpsLookupCacheLimit = 10
+	UINT64_MAX_VALUE    = ^uint64(0)
```

Comment:
what is this carat symbol before uint

Reply:
this is the bitwise XOR
every 0 bit gets set to 1

all 0's becomes all 1's aka the max value